libraries/chain/webassembly/runtimes/eos-vm.cpp

```diff
@@ -129,37 +130,67 @@ class eos_vm_instantiated_module : public wasm_instantiated_module_interface {
          _runtime(runtime),
          _instantiated_module(std::move(mod)) {}
 
+      void do_sync_call(apply_context& context) override {
+         backend_t                                bkend;
+         typename eos_vm_runtime<Impl>::context_t exec_ctx;
+         vm::wasm_allocator                       wasm_alloc;
+
+         const std::optional<sync_call_context> sync_call_ctx  = context.get_sync_call_ctx();
```

Comment:
Seems expensive to make this call of the sync call data.

Reply:
https://github.com/AntelopeIO/spring/pull/1251/commits/d031917abdc6f9ec8152f5edcf43581f36e47bca